.travis/script.sh

```diff
@@ -56,6 +58,9 @@ else
     # upload coverage report to codecov
     codecov
   fi
+
+  cat ~/resource_manager.log
+  cat ~/reserved_worker-1.log
```

Comment:
It looks like these lines are being called twice (although `reserved_workers-1.log` has changed to `reserved_worker-1.log` in your pr).

Reply:
They were. It's fixed now ty.